.github/workflows/stable.yml

```
@@ -145,20 +115,17 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ env.TOOLCHAIN }}
-          target: x86_64-pc-windows-msvc
+          target: ${{ matrix.target }}
           override: true
 
-      - name: Install dependencies
-        run: choco install nasm
-
-      - name: Run cargo test
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --target=x86_64-pc-windows-msvc --verbose
+      - name: Run cargo build
+        run: cargo build --verbose --target ${{ matrix.target }}
```

To be consistent with the other builds this should use the proper action:
```suggestion
        uses: actions-rs/cargo@v1
        with:
          command: build
          args: --target=${{ matrix.target }} --verbose
```